nodejs/week2/routes/documentRoute.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const Document = require("../controllers/documentController");
+
+router.route("/search").get(Document.getDocumentsByQuery);
+
+router.route("/:id").get(Document.getDocumentById);
+
+router.route("/search").post(Document.getDocumentsByFieldsOrQuery);
```

Comment:
It is a good rule to keep endpoints together, like
route("/:id").get
route("/search").get
route("/search").post

Reply:
Makes sense, I had ti grouped while debugging an issue, I changed it and forgot to update it. Updated now